ddprof-lib/src/main/cpp/wallClock.cpp

```diff
@@ -163,6 +163,19 @@ void WallClockJVMTI::timerLoop() {
   if (!isEnabled()) {
     return;
   }
+
+  jvmtiEnv* jvmti = VM::jvmti();
+  if (jvmti == nullptr) {
+    return;
+  }
+
+  jint threads_count = 0;
+  jthread* threads_ptr = nullptr;
+  if (jvmti->GetAllThreads(&threads_count, &threads_ptr) != JVMTI_ERROR_NONE ||
+    threads_count == 0) {
```

Comment:
Not my domain, though I think returning on threads_count == 0 is a functional difference.
I think it is OK to stop the profiler if there are no threads.